_ingest-pipelines/processors/fail.md

```diff
@@ -0,0 +1,171 @@
+---
+layout: default
+title: Fail
+parent: Ingest processors
+nav_order: 100
+---
+
+# Fail processor
+
+The `fail` processor can be useful for testing and debugging log ingestion pipelines. It allows you to intentionally fail the pipeline execution and observe the behavior or error handling mechanisms in your log ingestion and analysis systems. This can help you identify and resolve issues related to pipeline configuration, data transformation, or other aspects of log processing.
+
+The following is the syntax for the `fail` processor:
+
+```json
+"fail": {
+  "message": "Custom error message"
+}
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `fail` processor.
+
+Parameter | Required/Optional | Description |
```

Comment:
@gaobinlong Will you add the parameters? Don't worry about polishing the writing, I'll handle it. I just need the information. Thank you :)